crates/db/src/commands.rs

```diff
@@ -52,6 +52,14 @@ pub async fn db_get_contracts(chain_id: u32, db: tauri::State<'_, DB>) -> Result
     db.get_contracts(chain_id).await
 }
 
+#[tauri::command]
+pub async fn db_get_contracts_with_name(
```

Comment:
Do we really need to specify the "name" part? What about just using the previous function?

Reply:
I'll have to double check but I think the other function was used somewhere else and changing it broke something. I'll let you know after I look into this again